src/Popup.jsx

```diff
@@ -51,7 +51,7 @@ class Popup extends React.Component {
             <EventCell key={idx}
               {...props}
               event={event}
-              component={eventComponent}
+              component={components.event}
```

Comment:
we don't actaully want to pass the `components` prop through to this component, Instead can we fix why eventComponent wasn't being sent correctly?